server/errorHandler.ts

```diff
@@ -20,9 +20,39 @@ export default function createErrorHandler(production: boolean) {
       if (err.status === 403 && (!err.external || err.response?.body?.accessErrors)) {
         res.status(403)
 
+        const userHeaderService = 'You do not have permission to view this service'
+        const userHeaderPage = 'You do not have permission to view this page'
+        const returnedError = err.response?.body?.message
+        let userMessage
+        let userHeader
+
+        if (returnedError === 'could not map service provider user to access scope') {
+          userMessage =
+            'Your account is not set up correctly. Ask an admin user in your organisation to fix this in HMPPS Digital Services.'
+          userHeader = userHeaderService
+        } else if (returnedError === 'user is not a service provider') {
+          userMessage =
+            'Your account is not set up correctly. Ask an admin user in your organisation to fix this in HMPPS Digital Services.'
+          userHeader = userHeaderPage
+        } else if (returnedError === 'cannot find user in hmpps auth') {
+          userMessage =
+            'Your email address is not recognised. If it has changed recently, try signing out and signing in with the correct one. Ask an admin user in your organisation to check what the right email is in HMPPS Digital Services. If that does not work, <a target="_blank" href="https://hmpps-interventions-ui-dev.apps.live-1.cloud-platform.service.justice.gov.uk/report-a-problem">report it as a problem.</a>'
+          userHeader = userHeaderPage
+        } else if (returnedError?.includes('unidentified provider')) {
+          userMessage =
+            'Your provider group is not recognised. Ask an admin in your organisation to check it has been set up correctly in HMPPS Digital Services. They may need to <a target="_blank" href="https://hmpps-interventions-ui-dev.apps.live-1.cloud-platform.service.justice.gov.uk/report-a-problem">report it as a problem.</a>'
```

Comment:
> href="https://hmpps-interventions-ui-dev.apps.live-1.cloud-platform.service.justice.gov.uk/report-a-problem"

As-is, this link will be used in production as well -- can we use relative links or use the environment-specific host?

Reply:
Ah yes sorry my mistake I just copied directly from the requirements! can change this